Curricula/src/main/java/com/revature/models/Category.java

```diff
@@ -11,7 +11,6 @@
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 
 @Entity
-@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})// put this on all of your entities
```

Comment:
Why was this removed

Reply:
It was removed because it didnt need to be there. I can add it back in